provisioning/miniprov/create-template.sh

```diff
@@ -23,9 +23,9 @@ FILENAME=$1
 URL="json://$(readlink -f "$FILENAME")"
 
 conf "$URL" set 'cluster>my-cluster>site>storage_set_count=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>data=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>parity=0'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>spare=0'
+conf "$URL" set 'cluster>my-cluster>storage_set[0]>sns>data=1'
+conf "$URL" set 'cluster>my-cluster>storage_set[0]>sns>parity=0'
+conf "$URL" set 'cluster>my-cluster>storage_set[0]>sns>spare=0'
```

Comment:
I think similarly we also need to mention for >dix> as well here and modified references in other files

Reply:
Then we come up to the question how many options should we show in the templates. We're not given with any requirement on them.

Reply:
ok
I think here is what I understand from today's call:
For SNS (w.r.t. https://jts.seagate.com/browse/EOS-16763, but I am not sure if this is correct to refer here): 
data_units: 4
parity_units: 2
spare_units: 2

For DIX (https://jts.seagate.com/browse/EOS-18999 after knowing this requirement from Madhav): 
data_units: 1
parity_units: 1
spare_units: 1

For MD : My understanding from today's call is that, MD pool is going to be obselete soon. So, this PR wise, I think we can just have that pool type enum MD maintained, but we need not populate any value for MD pool now and we need to assume that it will not be there in ConfStore for now.

But please somebody confirm on my my understanding.
cc: @mssawant @s-arya

Reply:
Addressed in the new commit. Please review @suvratjoshi

Reply:
Not sure if I am missing something.
I was thinking, along with updates to values of layout N,K,S we also need to have >dix> too similar to sns.